### ISSUE TITLE
How to build and generate the APK?

### ISSUE THREAD

**Issue by Eric9453** [StevenSalazarM/Termux-api-bluetooth#3]
Hi StevenSalazarM,
I really enjoy this Termux-api-bluetooth project. Thank you very much.
However I don't know how to emerge this project to the newest termux apk (ver. 0.102).
I tried to copy termux-api.c to /data/data/com.termux/files/usr/libexec/, copy termux-bluetooth-connect and termux-bluetooth-scaninfo to /data/data/com.termux/files/usr/bin/. This doesn't work.

Can you teach me how to finish it? Many thanks.

**Comment by StevenSalazarM** [comment]
Yeah I tried to do the same with the newest version of termux app but it didn't work. However, from the old version I just had to change a [line](https://github.com/StevenSalazarM/termux-app-bluetooth/blob/eaed204c654b033c9daa8fd2ff4bc002614f10fb/app/src/main/java/com/termux/app/TermuxInstaller.java#L193) to make Termux-app download this instead of the [official bootstrap zip](https://github.com/termux/termux-app/blob/068cb7da500158f4c33b061376df15e96518feba/app/src/main/java/com/termux/app/TermuxInstaller.java#L173) (android-5 branch).

In the newest version I couldn't find anything similar to  determineZipUrl() so I think that once you find from where (and how) they are getting the bootstrap zip you can make it download your own version.
